### DISCUSSION TITLE
Classify coniferous and deciduous forest

### DISCUSSION THREAD

**Discussion by AndresKasekamp** [r-lidar/lidR#455] [Q&A] [answered]
Does the package include models to classify coniferous and deciduous forest stands? If not, could it be created? I have LiDAR datasets from spring and summer with a temporal variability of 2 years.

**Comment by Jean-Romain** [comment, marked as answer]
The package does not include any model of any kind. Models are study specific and are not generically applicable. Instead the package allows you to compute any kind of metrics and consequently allows to reproduce many models.
